src/v2/Apps/Artwork/Components/ArtistInfo.tsx

```diff
@@ -109,170 +98,150 @@ export class ArtistInfo extends Component<ArtistInfoProps, ArtistInfoState> {
-                meta={this.props.artist.formatted_nationality_and_birthday}
-                imageUrl={imageUrl}
-                href={this.props.artist.href}
-                FollowButton={
-                  <FollowArtistButton
-                    artist={this.props.artist}
-                    user={user}
-                    trackingData={{
-                      modelName: Schema.OwnerType.Artist,
-                      context_module: Schema.ContextModule.Biography,
-                      entity_id: internalID,
-                      entity_slug: slug,
-                    }}
-                    onOpenAuthModal={() =>
-                      this.handleOpenAuth(mediator, this.props.artist)
-                    }
-                    render={({ is_followed }) => {
-                      return (
-                        <Text
-                          data-test="followButton"
-                          style={{
-                            cursor: "pointer",
-                            textDecoration: "underline",
-                          }}
-                        >
-                          {is_followed ? "Following" : "Follow"}
-                        </Text>
-                      )
-                    }}
-                  />
-                }
-              />
-              {showArtistBio && (
-                <>
-                  <Spacer mb={1} />
-                  <ArtistBio
-                    bio={this.props.artist}
-                    onReadMoreClicked={this.trackArtistBioReadMoreClick.bind(
-                      this
-                    )}
-                  />
-                </>
-              )}
-              {showArtistInsightsButton && (
-                <Flex flexDirection="column" alignItems="flex-start">
-                  <Button
-                    onClick={
-                      this.state.showArtistInsights
-                        ? this.closeArtistInsights.bind(this)
-                        : this.openArtistInsights.bind(this)
-                    }
-                    variant="secondaryGray"
-                    size="small"
-                    mt={1}
-                  >
-                    {buttonText}
-                  </Button>
-                </Flex>
-              )}
-            </StackableBorderBox>
-            {this.state.showArtistInsights && (
-              <>
-                <ArtistMarketInsights
-                  artist={this.props.artist}
-                  border={false}
-                  Container={Container}
-                />
-                <SelectedExhibitions
-                  artistID={this.props.artist.internalID}
-                  border={false}
-                  totalExhibitions={this.props.artist.counts.partner_shows}
-                  exhibitions={this.props.artist.exhibition_highlights}
-                  ViewAllLink={
-                    <a
-                      href={`${sd.APP_URL}/artist/${this.props.artist.slug}/cv`}
+      <>
```

Comment:
What do these empty brackets do

Reply:
React requires a single dom element, but this statement returns more, `StackableBorderBox` and sometimes the insights components. Empty brackets are a syntax that allows wrapping jsx content without adding any HTML to the dom.